decidim-core/app/cells/decidim/coauthorships_cell.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This cell renders a collapsible list of a model's coauthors.
+  #
+  # Available sizes:
+  #  - `:small` => collapses after 3 elements.
+  #  - `:default` => collapses after 7 elements. If not specified, this one is
+  #    used.
+  #
+  # Example:
+  #
+  #    cell("decidim/coauthorships", @proposal)
+  class CoauthorshipsCell < Decidim::ViewModel
+    include Decidim::ApplicationHelper
+
+    def show
+      if model.respond_to?(:official?) && model.official?
```

Comment:
`if model.try(:official?)`

Reply:
I don't like to use `try` here because it suggests that the model should have this method but the model can be nil.

The longer expression expresses best the duck typing I guess.

Reply:
👍

Reply:
I've extracted the logic into a method.